test/int/stackTrace.test.ts

```diff
@@ -48,7 +48,9 @@ puppeteerSuite('Stack Traces', TEST_SPEC, (suiteContext) => {
             expectedFrames: [
                 { name: '(anonymous function)', line: 11, column: 9, source: { fileRelativePath: 'app.js' }, presentationHint: 'normal'},
                 { name: 'evalCallback', line: 12, column: 7, source: { fileRelativePath: 'app.js' }, presentationHint: 'normal'},
-                { name: '(eval code)', line: 1, column: 1, source: { evalCode: true }, presentationHint: 'normal'},
+                ((testSetup.isThisV2) ? // V1 produces "(anonymous function)" here
+                    { name: '(eval code)', line: 1, column: 1, source: { evalCode: true }, presentationHint: 'normal'} :

```

Comment:
Would it be better to extract 'eval code' vs 'anonymous function' to a variable at the top, and use that everywhere?

Reply:
Well, the challenge with that here is that it's not just 'eval' code vs 'anonymous function' here, but a series of strings/regexes all of which are slightly different, e.g.:

Line 51: (eval code) vs (anonymous function)
Line 75: "/\(eval code\) \[.*VM.*]/" vs "/\(anonymous function\) \[.*VM.*]/"
Line 98: "/\(eval code\) \[.*VM.*]/" vs "(anonymous function) Line 1"
Line 126: "/\(eval code\) \[.*VM.*]/" vs "/\(anonymous function\) \[.*VM.*] Line 1/"

So we'd need to create 4 constants for each of the sites. Easier to see the differences in-line IMO

Reply:
Resolved